File: Programmers/Lv1/Lv1_완주하지못한선수.py
def solution(participant, completion):
   participant.sort()
   completion.sort()
   for p, c in zip(participant, completion):
      if p != c:
         return p
   return participant[-1]

participant = ["mislav", "stanko", "mislav", "ana"]
completion = ["stanko", "ana", "mislav"]
print(solution(participant, completion))

File: Programmers/Lv1/Lv1_완주하지못한선수.cpp
#include <string>
#include <vector>
#include <set>
using namespace std;

string solution(vector<string> participant, vector<string> completion) {
	string answer = "";
	multiset<string> s;

	multiset<string>::iterator it;
	int ptcpt_size = participant.size();

	for (int i = 0; i < ptcpt_size; i++)
		s.insert(participant[i]);

	for (int i = 0; i < ptcpt_size - 1; i++) {
		it = s.find(completion[i]);
		s.erase(it);
	}

	it = s.begin();
	answer = *it;

	return answer;
}